ci/parameters.yml

```diff
@@ -0,0 +1,11 @@
+build-interval: 24h
+repository-url: https://github.com/GoogleCloudPlatform/gcsbeat
+# You will need the following secrets stored in credhub:
+# test-gcp-bucket-id - the name of the GCP bucket to run integrations with
```

Comment:
Consider improving the readability of this credhub param list by prefixing each item with " * " or something similar, just to make sure you catch the reader's eye and make it clear that this is a list.

Reply:
In the comment docs here I mean :)